resources/views/races/index.blade.php

```diff
@@ -24,52 +24,75 @@
     </nav>
     <main class="container">
         <section class="filter">
-            <p class="filter__header">Фильтры</p>
-            <div class="filter__option">
-                <div class="filter__titleBlock">
-                    <span class="filter__title">Город</span>
-                    <span class="filter__expand">+</span>
-                </div>
-                <div class="filter__cities">
-                    <div class="filter__cityOption">
-                        <input type="checkbox" id="moscow" name="cities">
-                        <label for="moscow">Москва</label>
-                    </div>
-                    <div class="filter__cityOption">
-                        <input type="checkbox" id="petersburg" name="cities">
-                        <label for="petersburg">Санкт-Петербург</label>
+            <form action="{{route('filterRace.show')}}" method="get">
+                <p class="filter__header">Фильтры</p>
+                <div class="filter__option">
+                    <div class="filter__titleBlock">
+                        <span class="filter__title">Город</span>
+                        <span class="filter__expand">+</span>
                     </div>
-                    <div class="filter__cityOption">
-                        <input type="checkbox" id="kazan" name="cities">
-                        <label for="kazan">Казань</label>
+                    <div class="filter__block">
+                        <div class="filter__item">
+                            <input type="checkbox" id="moscow" name="id city1" value="1">
+                            <label for="moscow">Москва</label>
+                        </div>
+                        <div class="filter__item">
+                            <input type="checkbox" id="petersburg" name="id city2" value="2">
+                            <label for="petersburg">Санкт-Петербург</label>
+                        </div>
+                        <div class="filter__item">
+                            <input type="checkbox" id="kazan" name="id city3" value="3">
+                            <label for="kazan">Казань</label>
+                        </div>
```

Comment:
Нужно выводить реальный список городов из базы. Что-то типа:
```suggestion
                        @foreach($cities as $city)
                            <div class="filter__item">
                                <label><input type="checkbox" name="city[]" value="{{ $city->id }}">
                                {{ $city->city }}</label>
                            </div>
                        @endforeach
```

(я пока не совсем понимаю, как ты реализовал фильтрацию, так что именно тот вариант, который я здесь написал, вряд ли будет работать)

Reply:
Не согласен)

Сейчас я захардкодил три города, и ручками поправил б\д, чтобы id соответствовали. Если мы добавим динамический вывод всех городов из б\д, у нас будет фильтрация из потенциально нескольких десятков пунктов, а это по-моему не особо круто.

Reply:
То есть проблема в том, что пользовательский интерфейс с большим количеством чекбоксов будет неудобный? Мне кажется, что эта проблема гораздо меньше, чем тот факт, что система фильтрации работает не с реальными данными.

Reply:
Да, проблема именно в этом) И я не совсем понимаю что означает утверждение, что система работает не с реальными данными, если они работает с базой данных?

Reply:
Это означает, что список городов захардкожен, то есть при изменении списка городов в базе это не отразится в фильтрах. То есть система работает не с реальным списком городов, а с каким-то своим, отличным от того, что в базе.

Если посмотреть на то, как подобная фильтрация делается в каких-нибудь интернет-магазинах, то там отображается несколько (видимо, наиболее популярных) вариантов и есть ссылка, по которой JS выдаёт полный список чекбоксов. Можно, конечно, и здесь организовать то же самое, но это сложнее и вряд ли входит в текущее задание. По-моему, лучше просто вывести столько чекбоксов, сколько нужно, и не заморачиваться пока слишком сильно про эстетику. (Если это сильно смущает, то можно изменить `CitySeeder`, чтобы создавать меньше городов.)

Reply:
И более того, идентификаторы городов тоже захардкожены, то есть фильтр по Москве отфильтрует на самом деле не Москву, а город с идентификатором 1, что бы это ни был за город

Reply:
В принципе, я с тобой согласен. Все что ты говоришь логично) Но речь идёт о том, что мы разрабатываем MVP. И для нас главное показать работающую функциональность, а не создавать оптимальное решение. Безусловно мой код можно оптимизировать и нужно это делать. Но на данном этапе он работает. А оптимизацию можно запилить в следующий спринт.